src/state/transactions/hooks.tsx

```diff
@@ -46,13 +49,20 @@ export function useTransactionAdder(): (
       if (!chainId) return
 
       const { hash } = response
+
+      const tx = await library?.getTransaction(hash)
```

Comment:
Will this throw error, causing we cant dispatch `addTransaction`?

Reply:
Should we wrap this into a try catch?

Reply:
no need. It will return null if it's not existed

Reply:
r u sure?